net/trafshow/Portfile

```diff
@@ -34,7 +34,7 @@ pre-configure {
 	xinstall -m 644 -W ${prefix}/share/libtool/build-aux config.guess config.sub ${worksrcpath}
 }
 
-configure.cflags-append   "-I${prefix}/include"
+configure.cflags-append   "-I${prefix}/include -Wno-implicit-function-declaration"
```

Comment:
we don't want to ignore the error; instead you should add the missing include at an appropriate place

Reply:
Fixed the other way around